src/views/Home/components/Landing/index.tsx

```diff
@@ -37,11 +37,8 @@ const Landing: React.FC = () => {
                 problems, pursue inclusion in tech, and kindle the spirit of
                 innovation.
               </div>
-              <div className='landing-component__location'>
-                &#47;&#47; location coming soon
-              </div>
               <div className='landing-component__dates'>
-                &#47;&#47; dates coming soon
+                January 14-16, 2022 &#160;&#183;&#160; Virtual
```

Comment:
Just to confirm, did you intend to put two spaces before and after the bullet point?

Also, I know I wrote the `&#47;` but this is actually getting confusing. I think we could either replace these with the HTML entity names (i.e.: `&nbsp;&middot;&nbsp;`), or just write them as unicode characters like the copyright symbol.

Reply:
After looking at this code again, I agree with on this and I'll replace the naming! After checking with design, we are intending for it to have 2 spaces before and after.